### DISCUSSION TITLE
Israel: Successfully enabled VoWiFi on Partner and Pelephone (Pixels 7 and 8 Pro)

### DISCUSSION THREAD

**Discussion by dvirgassner** [kyujin-cho/pixel-volte-patch#295] [Carrier-specific Experiences]
Successfully enabled voWiFi (voLTE already worked without this patch) on the **Partner** and **Pelephone** networks in Israel, in both a Pixel 8 Pro and a Pixel 7. Both carriers seem to natively support voLTE on these models but NOT voWiFi (as confirmed by their senior support staff).

**What did the trick is probably the addition of the "IMS" APN as explained [here](https://github.com/kyujin-cho/pixel-volte-patch/issues/136#issuecomment-1565598716).** 

I also did reset the network and carrier settings before adding the APN, as per the first troubleshooting suggestion [here](https://github.com/kyujin-cho/pixel-volte-patch/blob/main/docs/troubleshooting.en.md), but I doubt that made any difference.

Finally, note that voLTE will NOT work when you're connected to WiFi, only voWiFi will. This looks like natural behavior, since these features are mutually exclusive at any single time. Once I disconnected from WiFi, I could see that the status of voLTE in *#*#4636#*#* became "available," and voWiFi became "unavailable" -- and vice versa.

**Comment by Tom100pc** [comment]
HI, I'd like to buy a pixel 8 Pro (or 9 Pro XL) here in Israel. I have a Golantelecom account.
Will I be able to use the phone with voLTE? Golan can't tell me if a Pixel phone will work OK with them.
